_posts/2019-09-06-kubernetes-and-hokusai.md

```diff
@@ -0,0 +1,146 @@
+simple. Need logs? `hokusai production logs`. Or a review app? There are a
+[few steps involved](https://github.com/artsy/hokusai/blob/master/docs/Review_Apps.md), but you can have a
+fully-featured copy of your app up and running in a few minutes.
+
+The end of this post has a larger cheatsheet for handy Hokusai commands, but for now, let's talk about how you can
+use it yourself.
+
+# How can I set up Hokusai with my project?
+
+I should begin by noting that Hokusai is currently only meant to work with AWS - if your application is running on
+a different provider, you might have to hold off on Hokusai for now :( (or
+[open a PR in Hokusai](https://github.com/artsy/hokusai) yourself!)
+
+Installing hokusai is super easy! You can see full instructions in the README on
+[GitHub](https://github.com/artsy/hokusai), but if you're already set up with Python, pip, Docker, Docker Compose,
+and Git, you can do a quick install with Homebrew:
+
+```
+$ brew tap artsy/formulas
+$ brew install hokusai
+```
+
+There's more robust directions
+[in the Hokusai repo](https://github.com/artsy/hokusai/blob/master/docs/Getting_Started.md), but the very short
+version is that `hokusai setup` to handle most of the basics (creation of a Dockerfile, a config folder, and a few
+other bits and bobs). From there, you can customize according to the needs of your project.
+
+You should also check out Ash's [great post](https://artsy.github.io/blog/2018/01/24/kubernetes-and-hokusai/) on
+setting up a new Hokusai project - he runs through the process of setting up a new Rails application with Hokusai
+in an easy-to-follow way that also details small hitches he ran into along the way.
+
+# What's next for Hokusai?
+
+As Hokusai has grown and changed over the years (the GH repo was created in November 2016!), a few things have
+changed.
+
+For one, it's been increasingly used in coordination with CircleCI. Hokusai has made it really easy to standardize
+a lot of application configuration across Artsy's applications. We have
+[CircleCI orbs](https://github.com/artsy/orbs/blob/master/src/hokusai) set up for Hokusai specifically, which allow
+us to maintain one copy of our Hokusai config across multiple applications utilizing CircleCI. Given how helpful
+it's been to have a single source of CircleCI config for many of our apps, we're pondering the idea of a central
+source for Kubernetes Hokusai config. In other words, we'd like to have a "baseline" for things like deployments -
+something that could be overriden as necessary in specific projects but would make spinning up new projects easy.
+And it would hopefully make it easy to update deployment strategies for many apps at the same time...we'll see
+where we land.
+
+Interested in contributing? Hokusai is
+[open source](https://artsy.github.io/blog/2019/04/29/how-did-artsy-become-oss-by-default/) and we'd love to hear
+from you!
+
```

Comment:
Suggestion:

We've found Hokusai useful in transitioning our engineering team to working with Kubernetes, while easing the learning curve of working with it directly.  If you or your organization are going through similar growing pains, we suggest giving it a try!  Our issues are open for bug reports and feature requests and PRs open for improvements.